src/cities/CityConfig.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+class CityConfig {
+
+  cmsName: string
+  hostName: string
+  formsEnabled: boolean
+  title: string
+  logo: string
+  impressumUrl: string
```

Comment:
Could we have a title for the name of the Wohnngsbörse? like 'Wohnungsbörse Neuburg-Schrobenhausen' or smth like that?

Reply:
Sure, I'll add it.

Reply:
You can now use the `title` as requested.

Reply:
We don't need the impressumUrl  though.

Reply:
Even if we plan to supply it with an api?

Reply:
Yes, we only show the usual (integreat-like) "cms-internal" disclaimers, so we don't need external urls.

Reply:
I removed the impressumURL :)